packages/ai/src/types/schema.ts

```diff
@@ -77,10 +83,10 @@ export interface SchemaParams extends SchemaShared<SchemaInterface> {}
  */
 export interface SchemaRequest extends SchemaShared<SchemaRequest> {
   /**
-   * The type of the property. {@link
-   * SchemaType}.
+   * The type of the property. this can only be undefined when using `anyof` schemas,
```

Comment:
nit: `anyOf` capitalization to match the property capitalization.

Reply:
Done